build_local.sh

```diff
@@ -8,6 +8,13 @@ set -o errexit -o pipefail
 # Fail quickly if docker isn't working / up
 docker ps
 
+# Also fail quickly if Docker doesn't have enough memory to build Mesos
+DOCKER_MEMORY=$(docker info | grep Memory | awk '{print $3}')
+if (( $(echo "$DOCKER_MEMORY < 5" | bc -l) )); then
```

Comment:
I like it in general / everywhere. Linux hosts would also fail with too little RAM in fun mysterious ways (The nodejs / npm build of the UI also uses a ton of RAM)